tests/unit/helpers/format-message-test.js

```diff
@@ -160,7 +160,8 @@ test('able to discover all register translations', function(assert) {
 
 test('should respect format options for date ICU block', function(assert) {
   assert.expect(1);
-  this.render(hbs`{{format-message (l 'Sale begins {day, date, shortWeekDay}') day=1390518044403}}`);
+  this.day = 1390518044403;
```

Comment:
The reason for this change is that glimmer-vm has a bug in which VERY big integer throw an error: https://github.com/emberjs/ember.js/issues/15635